features/upgrade/svcat/upgrade.feature

```diff
@@ -0,0 +1,106 @@
+    Given the status of condition "Progressing" for "service-catalog-apiserver" operator is: False
+    Given the status of condition "Available" for "service-catalog-apiserver" operator is: True
+    Given the status of condition "Upgradeable" for "service-catalog-apiserver" operator is: Unknown
+    # Check cluster operator svcat-controller status
+    Given the "service-catalog-controller-manager" operator version matchs the current cluster version
+    Given the status of condition "Degraded" for "service-catalog-controller-manager" operator is: False
+    Given the status of condition "Progressing" for "service-catalog-controller-manager" operator is: False
+    Given the status of condition "Available" for "service-catalog-controller-manager" operator is: True
+    Given the status of condition "Upgradeable" for "service-catalog-controller-manager" operator is: True
+    #enable the svcat
+    Given I switch to cluster admin pseudo user
+    When I run the :patch client command with:
+      | resource      | ServiceCatalogAPIServer                 |
+      | resource_name | cluster                                 |
+      | p             | {"spec":{"managementState": "Managed"}} |
+      | type          | merge                                   |
+    Then the step should succeed
+    When I run the :patch client command with:
+      | resource      | ServiceCatalogControllerManager         |
+      | resource_name | cluster                                 |
+      | p             | {"spec":{"managementState": "Managed"}} |
+      | type          | merge                                   |
+    Then the step should succeed
+    
+  @admin
+  @upgrade-check
+  @users=upuser1,upuser2
+  Scenario: upgrade svcat
+    # Check cluster operator svcat-apiserver status
+    Given the "service-catalog-apiserver" operator version matchs the current cluster version
+    Given the status of condition "Degraded" for "service-catalog-apiserver" operator is: False
+    Given the status of condition "Progressing" for "service-catalog-apiserver" operator is: False
+    Given the status of condition "Available" for "service-catalog-apiserver" operator is: True
+    Given the status of condition "Upgradeable" for "service-catalog-apiserver" operator is: True
+    # Check cluster operator svcat-controller status
+    Given the "service-catalog-controller-manager" operator version matchs the current cluster version
+    Given the status of condition "Degraded" for "service-catalog-controller-manager" operator is: False
+    Given the status of condition "Progressing" for "service-catalog-controller-manager" operator is: False
+    Given the status of condition "Available" for "service-catalog-controller-manager" operator is: True
+    Given the status of condition "Upgradeable" for "service-catalog-controller-manager" operator is: True
+    
+    # Deploy ups broker
+    Given admin ensures "ups-broker" clusterservicebroker is deleted after scenario
```

Comment:
Similar thing, to avoid automation broken in clean up step, we have to identify `clusterservicebroker` resource exist before register this clean up step. 
Refer to https://github.com/openshift/verification-tests/pull/724/files#diff-b203d4aaff30ccc9f71afd12991e1b97R136
In another word, below steps should be added in L54:
```
    When I run the :get admin command with:
      | resource | clusterservicebroker |
    Then the step should succeed
```

Reply:
I guess we have implemented the same step in Line 96. So, I'd suggest that we remove this line or Line 96 part.

Reply:
@emmajiafan Please update it, thanks!

Reply:
done, add the check about "clusterservicebroker", remove the delete about "broker"